src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt

```diff
@@ -188,6 +188,11 @@ val orgNameCheck = Validation<String> {
     pattern(ORGANIZATION_NAME_REGEX) hint ORGANIZATION_NAME_REGEX_HINT
 }
 
+val nameOfUserCheck = Validation<String> {
```

Comment:
Do we need to add checks like
1. Min chars
2. Char count without spaces etc.?

Reply:
Done